apps/web/src/app/host-dashboard/page.tsx

```
@@ -0,0 +1,1131 @@
+            ? { ...p, status: p.status === 'active' ? 'inactive' : 'active' }
+            : p
+    ));
+};
+
+  const handleAddProperty = (e: { preventDefault: () => void; }) => {
+    e.preventDefault();
+    const property: Property = {
+      id: Date.now(),
+      title: newProperty.title,
+      location: newProperty.location,
+      price: parseInt(newProperty.price),
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)